src/game/client/vgui/client_viewport.cpp

```diff
@@ -311,6 +347,125 @@ void CClientViewport::HideScoreBoard()
 	m_pScorePanel->ShowPanel(false);
 }
 
+void CClientViewport::ShowHealthPanel()
+{
+	m_pHudHealthPanel->ShowPanel(true);
+}
+
+void CClientViewport::HideHealthPanel()
+{
+	m_pHudHealthPanel->ShowPanel(false);
+}
+
+void CClientViewport::IsHealthPanelVisible()
+{
+	if (m_pHudHealthPanel->IsVisible())
+	{
+		m_pHudHealthPanel->ShowPanel(false);
+	}
+	else
+	{
+		m_pHudHealthPanel->ShowPanel(true);
+	}
+}
```

Comment:
Function name is misleading (it toggles the visibility). And it looks like the function is not used, so you can just remove it.

Reply:
Done